python/task_list/console.py

```diff
@@ -0,0 +1,17 @@
+from typing import Optional, IO
+
+
+class Console:
+    def __init__(self, input_reader: IO, output_writer: IO) -> None:
+        self.input_reader = input_reader
+        self.output_writer = output_writer
+
+    def print(self, string: Optional="", end: str="\n", flush=True) -> None:
```

Comment:
Perhaps the type of `string` could be `Optional[str]` for increased safety.

Reply:
`flush` doesn’t have a type, which seems inconsistent with the rest of the function declaration.

Reply:
You're right. I'm still getting used to using typing in Python. I think it's a great addition, but as it is optional you can easily miss stuff. I'll make those changes now.